cve_bin_tool/checkers/kerberos.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/python3
+
+"""
+CVE checker for kerberos CLI
+
+References:
+
+RSS feed:
```

Comment:
Nitpick: this isn't an RSS feed.  We aren't using the RSS feeds currently so it doesn't have to be.  Could you just remove line 8?

Reply:
Thanks for the fix!